### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by flixoflax** [mtgibbs/chartist-plugin-barlabels#7]
Hey, 

I was wondering if it is possible to position the barlabel above the bar when the value is positive and below the bar when the value is negative.

My code for the bar chart options
`
    plugins: [
      Chartist.plugins.ctBarLabels({
        textAnchor: 'middle',
        labelClass: 'ct-label',
        showZeroLabels: true,
        labelOffset: {
          x: 0,
          y: -15
        },
      })
    ],
`

positions the label of positive values with an offset of -15  slightly above the bars.

When I have negative values it looks like this:
![image](https://user-images.githubusercontent.com/30297521/68544757-ddee2080-03c6-11ea-932a-9e80d0d3c52d.png)

I would like to put the labels below the bars, please consider to add an example code on how to achieve this:

![text842](https://user-images.githubusercontent.com/30297521/68544846-7e444500-03c7-11ea-9eb4-9a2c6f30dea8.png)

Best regards, 
flix

**Comment by mtgibbs** [comment]
Hmm...  I don't think I ever handled this use-case.  I'll have to sit down and work out a good solution.  There's likely a way to do this without the plugin using css classes, but that kind defeats the point.  Is that data-set private?  Can you post up the code that you're making that graph with and its data set?

**Comment by flixoflax** [comment]
Hey, thanks for the quick response.

This is my code:

`var data = {
    labels: ["Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Nov", "Dez"]
    series: [[6,0,1,-9,2,-46,-2,5,2,0,6,1],[1,2,6,-8,1,3,1,4,-1,57,1,0]]
  };
  var options = {
    textAnchor: 'middle',
    divisor: 4,
    seriesBarDistance: 10,
    chartPadding: {
      left: 40,
      top: 40,
      bottom: 40
    },
    axisX: {
      showGrid: false,
      showLabel: true,
      labelClass: 'ct-label',
      labelOffset: {
        x: 0,
        y: 15
      },
    },
    axisY: {
      showGrid: true,
      onlyInteger: true,
      labelClass: 'ct-label',
      labelOffset: {
        x: -15,
        y: 0
      },
      labelInterpolationFnc: function(value) {
        return value + '€';
      }
    },
    plugins: [
      Chartist.plugins.ctBarLabels({
        textAnchor: 'middle',
        labelClass: 'ct-label',
        showZeroLabels: true,
        labelOffset: {
          x: 0,
          y: -15
        },
      })
    ],
  };
  new Chartist.Bar('#bar-chart', data, options).on('draw', function(data) {
    if (data.type == 'bar') {
      data.element.animate({
        y2: {
          dur: '0.4s',
          from: data.y1,
          to: data.y2
        }
      });
    }
  });`

I could easily manage this problem if the dataset would be consistent but the dataset is changing all the time... so I need a solution that fits my needs.

**Comment by flixoflax** [comment]
Hey,

Were you able to solve my problem? ;)

Best regards

**Comment by mtgibbs** [comment]
No, I'm sorry.  I haven't had the time.  Holidays are always tight for me.  I wish you had of caught me last month.  I'll try to get this banged out, but the soonest I'd probably be able to take a look is two weekends from now.  :(

Thanks for your chart data though, I'll get to it when I can.

**Comment by flixoflax** [comment]
Alright. Thanks :)
